docs/tutorials/gemini-live-integration.mdx

```diff
@@ -0,0 +1,258 @@
+---
+type: tutorial
+sidebar_position: 5
+title: Gemini Live Integration
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Gemini Live Integration
+
+:::info
+This tutorial requires a working Fishjam backend. If you haven't set one up yet, please check the [Backend Quick Start](../tutorials/backend-quick-start).
+:::
+
+This guide demonstrates how to build a real-time speech-to-speech agent using Fishjam and [Google's Multimodal Live API](https://ai.google.dev/gemini-api/docs/live). By connecting these two services, you can create a low-latency voice assistant that listens to peers in a room and responds with natural voice.
+
+## Overview
+
+The implementation acts as a bridge between two real-time streams:
+
+1. **Fishjam ➡️ Gemini:** The Agent receives audio from the room and forwards it to Google GenAI.
+2. **Gemini ➡️ Fishjam:** The Agent receives audio generated by Gemini and plays it back into the room.
+
+To ensure these streams connect without audio glitches (garbled voice, wrong pitch), the audio sample rates must match perfectly between services.
```

Comment:
> sample rates must match perfectly between services.

`perfectly` sounds strange to me in this context. They just need to match, right?